prevents a git call that fails due to git ceiling set
diff --git a/meson.build b/meson.build
index 15fd5cd..6c1078d 100644
--- a/meson.build
+++ b/meson.build
@@ -7,7 +7,7 @@ project('gcab', 'c',
 
 git_version = []
 git = find_program('git', required: false)
-if git.found()
+if false
   git_version = run_command(git, 'describe', '--abbrev=4', '--dirty', check: true).stdout().strip().split('-')
 endif
 
